misc/scripts/update_GHpages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Following script copies the markdown files from vmware:master to gh-pages
+
+# get latest changes from master
+git pull
```

Comment:
nit: Please add some detail about the precondition ( project should have checked out, current state etc.)